cmd/helpMessages.go

```diff
@@ -323,6 +323,21 @@ Set the environment variable AZCOPY_SPA_CERT_PASSWORD to the certificate's passw
    Please treat /path/to/my/cert as a path to a PEM or PKCS12 file-- AzCopy does not reach into the system cert store to obtain your certificate.
    --certificate-path is mandatory when doing cert-based service principal auth.
 
+Log in using a Device:
+Set the environment variable AZCOPY_AUTO_LOGIN_TYPE=DEVICE to initiate Device login.
+   - azcopy login
+   Please note that you will be provided with a code to authenticate via a web browser. For example, you may see a message like this:
+   To sign in, use a web browser to open the page https://microsoft.com/devicelogin and enter the code AWRKVXE8P.
```

Comment:
Change the code to something generic - e.g. `ABCD12345`

Reply:
Done in the latest commit.